exp/query/api.go

```diff
@@ -87,7 +90,15 @@ func NewAPI(
 
 // Search serves vector search over the offloaded tenant on object storage.
 func (a *API) Search(ctx context.Context, req *SearchRequest) (*SearchResponse, error) {
-	info, tenantVersion, err := a.schema.TenantStatus(ctx, req.Collection, req.Tenant)
+	// Expectations
```

Comment:
I think we can remove this comment. It's too implementation oriented I think. probably you got back when rebasing your branch.